apps/crn-server/test/data-providers/working-groups.data-provider.test.ts

```diff
@@ -192,9 +190,13 @@ describe('Working Group Data Provider', () => {
         expect(response?.leaders).toStrictEqual([
           {
             user: {
-              ...getSquidexUserGraphqlResponse().findUsersContent,
-              ...getSquidexUserGraphqlResponse().findUsersContent?.flatData,
+              id: 'user-id-1',
```

Comment:
is this only a subset of a user from the `getSquidexUserGraphqlResponse()`? Maybe we should get these selected properties from the fixture rather than create a separate expectation fixture embedded in the test? 

I am only saying this since we had lots of issues with refactoring because of too many expectations and fixtures embedded in the tests. If you are going to do this and leave it embedded at least extract it into a const and make sure you assign it a type (this could be replaced with a [`satisfies` operator in TS 4.9](https://devblogs.microsoft.com/typescript/announcing-typescript-4-9/#the-satisfies-operator). That way should we change the expected output type at least TS will tell us where make the changes in our tests

Reply:
it makes sense, it was committed here https://github.com/yldio/asap-hub/pull/2359/commits/7bfcb1913f184170a90c7e1a0450111b1cd2438d

Reply:
@gabiayako was it though? this one is still hardcoded - i think you only changed the other one below

Reply:
sorry, it was lack of attention 🙈 now it has been fixed https://github.com/yldio/asap-hub/pull/2359/commits/ed8bca11b729bec973cc5be02d8ec1556d2a3330